src/components/classrooms/ImportDataButton.js

```diff
@@ -0,0 +1,48 @@
+import React, { Component } from 'react';
+import { withStyles } from '@material-ui/core';
+import { withTranslation } from 'react-i18next';
+import { withRouter } from 'react-router';
+import PropTypes from 'prop-types';
+import Button from '@material-ui/core/Button';
+
+const styles = () => ({
+  center: {
+    margin: 'auto',
+    display: 'block',
+  },
+});
+
+class ImportDataButton extends Component {
+  static propTypes = {
+    classes: PropTypes.shape({
+      center: PropTypes.string.isRequired,
+    }).isRequired,
+    t: PropTypes.func.isRequired,
+  };
+
+  handleOnClick = () => {
+    // todo: go to student data import page
```

Comment:
This is not needed now?

Reply:
No but it will be needed in the next PR.